### DISCUSSION TITLE
Tutorial 3 for GAT mistake

### DISCUSSION THREAD

**Discussion by yCobanoglu** [pyg-team/pytorch_geometric#7040] [General]
https://colab.research.google.com/github/AntonioLonga/PytorchGeometricTutorial/blob/main/Tutorial3/Tutorial3.ipynb
You define GATLayer but end up using GATConv from the library for the model. The layer is never used.

**Comment by rusty1s** [comment]
@AntonioLonga

**Comment by AntonioLonga** [comment]
Hi @yCobanoglu, 

yes, you are right. the idea is to implement a GAT layer manually, to show how to build ad hoc layers and then we used the original one. I think that If you have a look at the video, then it is easiest to follow the story. :)
